fixtures/test_configs.go

```diff
@@ -21,7 +21,8 @@ func SetupTomlTestConfig() *TestConfigs {
   host = "github.com"
   user = "jingweno"
   access_token = "123"
-  protocol = "http"`
+  protocol = "http"
+  unix_socket = "/tmp/go.sock"`
```

Comment:
This adds `unix_socket` to default test configs. Is this desired, or should we create separate test cases which exercise the unix socket functionality?

Reply:
Added a separate test config in 0eb76ff